_posts/2017-09-19-FAF-in-container.markdown

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title:  "FAF in container"
+date:   2017-09-19 10:00:00 +0100
+author: marusak
+visible: 1
+categories: FAF container
+---
+It has never been easier to deploy FAF.
+
+# Why deploying own FAF
+Firstly, standard users probably do not want to deploy own FAF - FAF is here
+to collect a huge amount of crashes from lots of users. Deploying own FAF might
+be interesting for SysAdmins who control hundreds of machines. It also may be
+very interesting on container platform such as OpenShift.
```

Comment:
Or for somebody who does not want to share his crashes publicly.

Reply:
I thought about that, but I intentionally haven't written that not to encourage people deploying own FAF, because they are scared of `leeking` privacy, and therefore devaluating point of public instances of FAF (fewer people reporting).